src/NServiceBus.AcceptanceTesting/ScenarioContext.cs

```diff
@@ -28,10 +29,17 @@ public void AddTrace(string trace)
 
         ConcurrentQueue<string> traceQueue = new ConcurrentQueue<string>();
 
+        internal Dictionary<string, LogLevel> LogLevels = new Dictionary<string, LogLevel>();
+
+        public void SetLogLevel(string logger, LogLevel level)
```

Comment:
tbh, the better solution would probably be to log all the things with loglevel, logger name and message and only configure the trace output?